tools/wagmi-example/README.md

```diff
@@ -0,0 +1,21 @@
+# Wagmi Hedera Example
+
+## How to run example?
+1. Install dependencies `pnpm install`
+2. Run project `pnpm dev`
```

Comment:
Is it possible to use `npm` instead of `pnpm`? We use `npm` all over the place.

Reply:
done